BunqSdk.Tests/Model/Generated/Object/NotificationUrlTest.cs

```diff
@@ -58,23 +60,32 @@ public class NotificationUrlTest : BunqSdkTestBase
 
         private void ExecuteNotificationUrlTest(
             string expectedJsonFileName,
-            Type classNameExpected,
-            string referencedObjectPropertyName
-            ) {
+            Type classTypeExpected,
+            string referencedObjectPropertyName,
+            string subClassObjectPropertyName = null,
+            Type subClassTypeExpected = null) {
```

Comment:
please lookup c# style conventions... I think `) {` needs to be on a new line.

Reply:
The SDK has the following structure:
```
        private BunqResponseRaw SendRequest(HttpRequestMessage requestMessage,
            IDictionary<string, string> customHeaders, string uriRelative)
        {
```

Only `{` on its own line without `)`

Reply:
bon!